src/app/demo/pagination-range/pagination-range.component.ts

```diff
@@ -42,7 +42,7 @@ export class PaginationRangeComponent implements OnInit {
 
   ngOnInit(): void {
     this.configuration = { ...DefaultConfig };
-    this.configuration.paginationMaxSize = 7;
+    this.configuration.rows = 7;
```

Comment:
I just checked this:
```ts
this.configuration.paginationMaxSize = 7;
this.configuration.rows = 3;
```
And this works fine. What is the issue here?

<img width="1414" alt="Screenshot 2021-11-08 at 10 03 34 AM" src="https://user-images.githubusercontent.com/6843074/140713575-45e538c3-e9dc-44e0-be6b-9bff3d683a9c.png">

Reply:
![image](https://user-images.githubusercontent.com/80253573/140745794-f5483a0e-5bc1-42ac-bd04-e67f4f351e31.png)

Here pagination size is 7 but still showing 10 records, 
** this.configuration.rows = 7;** - this line is missing in code

Reply:
Thanks, 
`paginationMaxSize` is a property where you define the maximum number of pages in the pagination. So when you set it to 3 it is like an image below. `rows` is a property to set an aomut of the rows in the table.

<img width="175" alt="Screenshot 2021-11-08 at 2 24 11 PM" src="https://user-images.githubusercontent.com/6843074/140749677-6f0566d6-2c3a-4190-bbda-bd8718839aee.png">
s

Reply:
Okay, thanks